shared/constants/network.ts

```diff
@@ -934,7 +934,7 @@ export const FEATURED_RPCS: AddNetworkFields[] = [
     nativeCurrency: CURRENCY_SYMBOLS.ETH,
     rpcEndpoints: [
       {
-        url: LINEA_MAINNET_RPC_URL,
+        url: `https://linea-mainnet.infura.io/v3/${infuraProjectId}`,
         type: RpcEndpointType.Custom,
```

Comment:
Sorry dnt see why are we not using `LINEA_MAINNET_RPC_URL` here?

Reply:
This was actually Brian's feedback from Slack. The idea was to keep the configuration as close to the other ones here in this configuration. The other linea related rpcs are string literals like this.